api/src/lib/inventory/groupvar.mjs

```diff
@@ -92,3 +92,26 @@ Groupvar.prototype.delete = async function (id = false) {
 
   return result
 }
+
+/**
+ * Helper method to delete a groupvar with group_id
+ *
+ * @param {string} group_id - The group_id of the record to delete
+ * @return {bool} result - true if it went ok, false if not
+ */
+Groupvar.prototype.deletes = async function (group_id = false) {
```

Comment:
Can we give this a more intuitive name?
We have `Groupvar.delete()` (which I can imagine what it does) and `Groupvar.deletes()` (which is a bit confusing).

Since this removes the groupvars with a given group id, perhaps something like `Groupvar.deleteGroupvars()`?